### DISCUSSION TITLE
UTM Ubuntu 22.04 screen lock when resize window manualy

### DISCUSSION THREAD

**Discussion by acrimu** [utmapp/UTM#4707] [UTM Help (macOS)]
Whenever I resize the WM window, Ubuntu is going in screen lock and I have to enter the password again.
I disabled screen lock from Ubuntu settings but it didn't help.
Is there any way to avoid this behavior?

**Comment by freebrowser1** [comment]
Same issue here, but it is probably not a UTM issue, but of Ubuntu itself.

**Comment by acrimu** [comment]
Probably not an UTM issue, but this behavior appears when resizing the VM.
I will give it a try in VmWare to see how it behaves.
It would be good to find a solution for this.

**Comment by chetbox** [comment]
Same problem here. I have auto login turned on and here are the screen lock settings in Ubuntu 22.04.2.

<img width="917" alt="image" src="https://user-images.githubusercontent.com/292958/232344619-28a81919-a9cb-4df6-9318-6f286e1dd066.png">

**Comment by rsmogura** [comment]
This is GDM3 related. When I switched to SDDM (KDE) no lock happened on resize.

**Comment by freebrowser1** [comment]
Even weirder, I have both Ubuntu 22.04 VMs, one ARM64 and one x86-64 on my Macbook M1.
The ARM Version has this problem, but the Intel version not.
Probably it is ARM related.

**Comment by tomfaulhaber** [comment]
Had this same problem but updating to the latest SW in Ubuntu 22.04 on 6/21/23 fixed it.

**Reply by faisalmustafa** [reply]
Hi,
It's great to hear that you found a solution. I am new here. Would you explain how to do that? What does SW stand for?

Thanks for your kind help.
